Sources/XCRemoteCache/Models/MetaWriter.swift

```diff
@@ -24,11 +24,16 @@ protocol MetaWriter {
 }
 
 class JsonMetaWriter: MetaWriter {
-    private let metaEncoder = JSONEncoder()
+    private let metaEncoder: JSONEncoder
     private let fileWriter: FileWriter
 
-    init(fileWriter: FileWriter) {
+    init(fileWriter: FileWriter, pretty: Bool = false) {
```

Comment:
Can we have non-optional paramters? This is a convention in XCRemoteCache. 
Side-effect: a need to change a unit test, will suggest writing an integration unit test.

```suggestion
    init(fileWriter: FileWriter, pretty: Bool) {
```